UI/CMakeLists.txt

```diff
@@ -325,7 +325,7 @@ set(obs_HEADERS
 	remote-text.hpp
 	audio-encoders.hpp
 	qt-wrappers.hpp
-	clickable-label.hpp)
+	clickable-label.hpp obs-proxy-style.cpp obs-proxy-style.hpp)
```

Comment:
The cpp file shouldn't be in the headers section. Also the files should be on separate lines.